src/stampit.js

```diff
@@ -114,7 +111,7 @@ function compose(factories) {
  * @return {Function(statics)} factory.static Add properties to the stamp (not objects!). Chainable.
  */
 stampit = function stampit(options) {
-  var fixed = {methods: {}, refs: {}, init: [], props: {}, static: {}};
+  let fixed = {methods: {}, refs: {}, init: [], props: {}, static: {}};
```

Comment:
I'm pretty sure `const` mean you cannot assign to this variable any more.

The following works well:
```js
const obj = { a: 1 };
obj.a = 2; // no errors :)
```
But the following is error:
```js
obj = {}; // ERROR!
```